spyder/widgets/editor.py

```diff
@@ -280,7 +280,7 @@ class EditorStack(QWidget):
     readonly_changed = Signal(bool)
     encoding_changed = Signal(str)
     sig_editor_cursor_position_changed = Signal(int, int)
-    refresh_eol_chars = Signal(str)
+    signal_refresh_eol_chars = Signal(str)

```

Comment:
We follow the convention of naming signals with the `sig_` prefix, not `signal_`